src/components/inputs/text/ValidationInput/ValidationInput.tsx

```diff
@@ -66,31 +78,90 @@ function ValidationInput({
     );
 
     setValidationResults(validationResults);
-    if (setIsValid) {
-      setIsValid(validationResults.every((value) => value.status === true));
+    const validity = validationResults.every((value) => value.status === true);
+    setValid(validity);
+    if (validityCallback) {
+      validityCallback(validity);
     }
   }
 
   return (
     <>
       {/* eslint-disable-next-line */}
       <div
+        id={'validation-input'}
         className={wrapperClassName}
         style={{ ...wrapperCustomStyle }}
         onClick={onClick ? () => onClick() : undefined}
       >
-        <input
-          id={inputId}
-          type="text"
-          className={inputClassName}
-          maxLength={maxLength}
-          placeholder={placeholder}
-          value={value}
-          onChange={(e) => validationExecutor(e.target.value)}
-          disabled={disabled}
-          style={{ ...inputCustomStyle }}
-        />
-
+        <div className="flex" style={{ width: '100%', position: 'relative' }}>
+          <input
+            id={inputId}
+            type="text"
```

Comment:
pass this as a prop, TTL seconds should be `number`

Reply:
added

Reply:
Not seeing the changes

Reply:
should be up now